Package.swift

```diff
@@ -0,0 +1,113 @@
+// swift-tools-version:5.3
+import PackageDescription
+import Foundation
+
+private struct Executable {
```

Comment:
Do I remember you saying this was inspired by something you found online? If so, a credit/source comment would be nice so we can trace where it came from.